### PR TITLE
Adding bigquery.metadataViewer role to SA

### PR DESCRIPTION
**Pull request by marko7460**
Adding `roles/bigquery.metadataViewer` to SA as requested by @hshin-g

### CODE REVIEW

File: test/integration/simple_example/controls/forseti.rb [hunk 1]
```diff
@@ -184,5 +185,38 @@
       end
     end
   end
+end
+
+control 'forseti-org-iam' do
```
Comment (adrienthebo) [inline test/integration/simple_example/controls/forseti.rb]: 💯 , thanks for adding this test coverage!

File: .kitchen.yml [hunk 2]
```diff
@@ -54,6 +54,12 @@ suites:
             - test/fixtures/simple_example/sshkey
           shell: true
           shell_options: "--login"
+
+        - name: org-iam
```
Comment (adrienthebo) [inline .kitchen.yml]: Right now these tests will only run on the `simple-example` test fixture, which means we have a gap for shared-vpc. We should address this but that change will be somewhat involved, so this is mainly a note for posterity.

File: test/integration/simple_example/controls/forseti.rb [hunk 3]
```diff
@@ -184,5 +185,38 @@
...
+  title "Validate organization roles of SA"
+  describe command("gcloud organizations get-iam-policy #{org_id} --filter='bindings.members:#{forseti_server_service_account}' --flatten='bindings[].members' --format='csv(bindings.role)'") do
```
Comment (adrienthebo) [inline test/integration/simple_example/controls/forseti.rb]: In most/all of our tests that shell out to `gcloud` we generally use the `json` format. Is there a particular reason why we chose CSV for this test?

Comment (marko7460) [reply test/integration/simple_example/controls/forseti.rb]: It was easier for me to list the roles in CSV and parse in ruby. https://github.com/marko7460/terraform-google-forseti/blob/feature/add_missing_role/test/integration/simple_example/controls/forseti.rb#L196-L201.
I can fix the code to deal with json if that is important for consistency with other projects and tests.

File: test/integration/simple_example/controls/forseti.rb [hunk 4]
```diff
@@ -184,5 +185,38 @@
...
+    its(:exit_status) { should eq 0 }
+    its(:stderr) { should eq '' }
+
+    let(:sa_roles) do
```
Comment (aaron-lane) [inline test/integration/simple_example/controls/forseti.rb]: This definition can be simplified.

```rb
let :sa_roles do
  CSV.parse(subject.stdout)[1..-1].map(&:first)
end
```

File: test/integration/simple_example/controls/forseti.rb [hunk 5]
```diff
@@ -184,5 +185,38 @@
...
+      roles = Array.new
+      CSV.parse(subject.stdout).each {|role|
+        roles.push(role[0])
+      }
+      roles[1..-1]
+    end
 
+    expected_roles = [
```
Comment (aaron-lane) [inline test/integration/simple_example/controls/forseti.rb]: This should be defined with `let :expected_roles`.

File: test/integration/simple_example/controls/forseti.rb (hunk 3 above)
Comment (adrienthebo) [reply test/integration/simple_example/controls/forseti.rb]: @marko7460 yeah, let's just switch this to JSON. We don't benefit from using CSV and it adds complexity and cognitive load; let's go with the common convention.

File: .kitchen.yml (hunk 2 above)
Comment (aaron-lane) [reply .kitchen.yml]: The tests have been added to the `shared_vpc` suite.

**Pull request merged**